subxt/src/client.rs

```diff
@@ -404,6 +398,56 @@ where
 
         // Wrap in Encoded to ensure that any more "encode" calls leave it in the right state.
         // maybe we can just return the raw bytes..
-        Ok(Encoded(extrinsic))
+        Ok(SignedSubmittableExtrinsic {
+            client: self.client,
+            encoded: Encoded(extrinsic),
+            marker: self.marker,
+        })
+    }
+}
+
+pub struct SignedSubmittableExtrinsic<'client, T: Config, X, E: Decode, Evs: Decode> {
+    client: &'client Client<T>,
+    encoded: Encoded,
+    marker: std::marker::PhantomData<(X, E, Evs)>,
+}
+
+impl<'client, T, X, E, Evs> SignedSubmittableExtrinsic<'client, T, X, E, Evs>
+where
+    T: Config,
+    X: ExtrinsicParams<T>,
+    E: Decode + HasModuleError,
+    Evs: Decode,
+{
+    /// Submits the extrinsic to the chain.
+    ///
+    /// Returns a [`TransactionProgress`], which can be used to track the status of the transaction
+    /// and obtain details about it, once it has made it into a block.
+    pub async fn submit_and_watch(
+        &self,
+    ) -> Result<TransactionProgress<'client, T, E, Evs>, BasicError> {
+        // Get a hash of the extrinsic (we'll need this later).
+        let ext_hash = T::Hashing::hash_of(&self.encoded);
+
+        // Submit and watch for transaction progress.
+        let sub = self.client.rpc().watch_extrinsic(&self.encoded).await?;
+
+        Ok(TransactionProgress::new(sub, self.client, ext_hash))
+    }
+
+    /// Submits the extrinsic to the chain for block inclusion.
+    ///
+    /// Returns `Ok` with the extrinsic hash if it is valid extrinsic.
+    ///
+    /// # Note
+    ///
+    /// Success does not mean the extrinsic has been included in the block, just that it is valid
+    /// and has been included in the transaction pool.
+    pub async fn submit(&self) -> Result<T::Hash, BasicError> {
+        self.client.rpc().submit_extrinsic(&self.encoded).await
+    }
+
+    pub fn encoded(&self) -> Encoded {
+        self.encoded.clone()
     }
```

Comment:
As suggested in the other comment, I reckon this can return a `&[u8]`, giving users the option to clone or not as needed. I'd add some docs too; something like:

```
/// Returns the SCALE encoded extrinsic bytes.
```